mne/channels/montage.py

```diff
@@ -14,6 +14,7 @@
 import os.path as op
 
 import numpy as np
+from pandas import read_csv
```

Comment:
you cannot do this. pandas is not a hard dependency of MNE. See how we do this in other places in the code.

Reply:
better yet, see this section of the contributor guide: https://mne-tools.github.io/dev/install/contributing.html#importing  :)